packages/gafl-webapp-service/src/pages/multibuy/view-licences/__tests__/route.spec.js

```diff
@@ -94,19 +105,55 @@ describe('view licences > getData', () => {
     })
 
     const getSampleRequest = () => ({
-      ...mockRequest,
+      ...mockRequestPermission,
       i18n: {
         getCatalog: () => catalog
+      },
+      url: {
+        search: ''
+      }
+    })
+
+    const mockRequestNoPermission = createMockRequest({
+      cache: {
+        transaction: {
+          permissions: [emptyPermission]
+        }
+      }
+    })
+
+    const getMockRequest = () => ({
+      ...mockRequestNoPermission,
+      i18n: {
+        getCatalog: () => catalog
+      },
+      url: {
+        search: ''
       }
     })
 
     beforeEach(() => jest.clearAllMocks())
 
-    it('returns permissions, licences and text for start after minutes text', async () => {
+    it('returns duplicate, licences, licences remaining being true and uri matching licence for', async () => {
       const res = await getData(getSampleRequest())
       expect(res).toMatchSnapshot()
     })
 
+    it('returns false duplicate, null licences, licences remaining being false and uri matching licence for', async () => {
+      const res = await getData(getMockRequest())
+      expect(res).toMatchSnapshot()
```

Comment:
in the snapshot, `licences` is null. One thing I forgot to mention in the confluence page was that in a snapshot, ideally all values should be populated. I don't see a test here for a populated licences property (I may have missed it).

Reply:
licences expected be null as removed all licences, test above has snapshot with licences with value

Reply:
Fair enough, my mistake